packages/dmn-js-decision-table/src/features/add-rule/PostAddRuleBehavior.js

```diff
@@ -0,0 +1,20 @@
+import CommandInterceptor from 'diagram-js/lib/command/CommandInterceptor';
+
+export default class PostAddRuleBehavior extends CommandInterceptor {
+  constructor(eventBus, selection) {
+    super(eventBus);
+
+    this.postExecuted('row.add', (event) => {
+
+      eventBus.once('elements.changed', 500, function() {
```

Comment:
Was there any particular reason to use arrow function in the first listener and then the `function` in the second?

Reply:
There isn't, that's an oversight on my end